src/components/application_manager/test/commands/mobile/perform_interaction_test.cc

```diff
@@ -236,6 +237,7 @@ TEST_F(PerformInteractionRequestTest, OnTimeout_VR_GENERIC_ERROR) {
   EXPECT_EQ(
       (*vr_command_result)[strings::msg_params][strings::result_code].asInt(),
       static_cast<int32_t>(am::mobile_api::Result::GENERIC_ERROR));
+  EXPECT_EQ((*vr_command_result)[strings::msg_params][strings::info].asString(), "");
```

Comment:
@VVeremjova fixed in https://github.com/smartdevicelink/sdl_core/pull/1368/commits/e1e998ffc82c12f956a5d10777d9061aa13daae2